### PR TITLE
Dynamic pages

### PR DESCRIPTION
**Pull request by J4ken**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -57,6 +57,9 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
+# nokogiri
```
Comment (ljukas) [inline Gemfile]: Move the un-grouped gems up to the other un-grouped gems.

Comment (ljukas) [inline Gemfile]: Move the un-grouped gems up to the other un-grouped gems.

File: app/controllers/contacts_controller.rb [hunk 2]
```diff
@@ -1,19 +1,19 @@
 class ContactsController < ApplicationController
 
   def press
-    @contacts = Contact.where(group: 'low')
```
Comment (ljukas) [inline app/controllers/contacts_controller.rb]: Changed names to english names instead of "huvudansvarig"

File: config/routes.rb [hunk 3]
```diff
@@ -2,17 +2,13 @@
...
-
   # Controller
   get 'contact' => 'contacts#contact'
   get 'press' => 'contacts#press'
 
-
   # Let’s encrypt
   get '/.well-known/acme-challenge/:id' => 'lets_encrypt#challenge', as: :letsencrypt_challenge
 
+  # Static controller
```
Comment (patriksletmo) [inline config/routes.rb]: This is not a static controller, please change.

File: test/fixtures/pages.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (patriksletmo) [inline test/fixtures/pages.yml]: Delete fixtures for deleted model.

File: test/models/page_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (patriksletmo) [inline test/models/page_test.rb]: Remove test.

File: test/models/info_test.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (patriksletmo) [inline test/models/info_test.rb]: Remove test.

File: test/fixtures/infos.yml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (patriksletmo) [inline test/fixtures/infos.yml]: Remove unused fixture.

File: db/seeds.rb [hunk 8]
```diff
@@ -16,26 +16,21 @@
...
- {name: "Victoria Boman", title: "Personal", email: "personal@sof17.se", group: "high"},
- {name: "Simon Rung", title: "Administration", email: "administration@sof17.se", group: "low"},
- {name: "Filip Renström", title: "Kampanj", email: "kampanj@sof17.se", group: "low"},
- {name: "Tobias Sjöberg", title: "servering", email: "servering@sof17.se", group: "low"},
- {name: "Anton Kullberg", title: "Område", email: "omrade@sof17.se", group: "low"},
- {name: "Filip Gardler", title: "SoS", email: "sos@sof17.se", group: "low"},
- {name: "Marcus Eriksson", title: "Säkerhet", email: "sakerhet@sof17.se", group: "low"},
- {name: "Junior Asante", title: "Kårtege", email: "kartege@sof17.se", group: "low"},
- {name: "Irma Hamzic", title: "MF", email: "mf@sof17.se", group: "low"}
+ {name: "Daniel Modig", title: "General", email: "general@sof17.se", group: "huvudansvarig"},
```
Comment (ljukas) [inline db/seeds.rb]: Change to english group-names

File: app/models/info.rb [hunk 9]
```diff
@@ -0,0 +1,2 @@
+class Info < ApplicationRecord
```
Comment (patriksletmo) [inline app/models/info.rb]: Remove model.

File: test/controllers/pages_controller_test.rb [hunk 10]
```diff
@@ -1,7 +0,0 @@
-require 'test_helper'
```
Comment (ljukas) [inline test/controllers/pages_controller_test.rb]: Why removed a test? (even if it doesnt test anything, good to have the base of a test)

File: db/schema.rb [hunk 11]
```diff
@@ -21,4 +21,23 @@
     t.string   "group"
   end
 
+  create_table "infos", force: :cascade do |t|
```
Comment (patriksletmo) [inline db/schema.rb]: Update schema after removing info model.

File: db/migrate/20170119110821_create_infos.rb [hunk 12]
```diff
@@ -0,0 +1,8 @@
+class CreateInfos < ActiveRecord::Migration[5.0]
```
Comment (patriksletmo) [inline db/migrate/20170119110821_create_infos.rb]: Make sure to delete migration when deleting info model.

File: test/controllers/pages_controller_test.rb (hunk 10 above)
Comment (J4ken) [inline test/controllers/pages_controller_test.rb]: does nothing jon snow

File: app/controllers/contacts_controller.rb (hunk 2 above)
Comment (J4ken) [inline app/controllers/contacts_controller.rb]: caaaaann doooo

File: test/fixtures/pages.yml (hunk 4 above)
Comment (J4ken) [inline test/fixtures/pages.yml]: borta

File: test/models/page_test.rb (hunk 5 above)
Comment (J4ken) [inline test/models/page_test.rb]: borta

File: test/models/info_test.rb (hunk 6 above)
Comment (J4ken) [inline test/models/info_test.rb]: borta

File: test/fixtures/infos.yml (hunk 7 above)
Comment (J4ken) [inline test/fixtures/infos.yml]: borta

File: db/schema.rb (hunk 11 above)
Comment (J4ken) [inline db/schema.rb]: done

File: db/migrate/20170119110821_create_infos.rb (hunk 12 above)
Comment (J4ken) [inline db/migrate/20170119110821_create_infos.rb]: sure

**Pull request merged**